### DISCUSSION TITLE
Start WIS as background process

### DISCUSSION THREAD

**Discussion by ceinstaller** [toverainc/willow-inference-server#106] [General]
Hey all, I'm showing my ignorance of docker here, but how do I start the server as a background process?  I tried:

`sudo ./utils.sh run &`

but it exited without starting the server.  I tried looking at the shell script but it's over my head.  Is there a way to fire it off using systemctl?  (I'm running it on Ubuntu Server 22.04)  For now I'm starting it in a putty session and just disconnecting.  It works but it would be cool to have it as a service.

Thanks in advance,

~Mark

**Comment by kristiankielhofner** [comment]
As of commit c15a15a you can use the `-d` argument to run which will run in the background and start automatically on boot:

`./utils.sh run -d`
